app/src/main/java/net/gsantner/markor/ui/fsearch/FileSearchResultSelectorDialog.java

```diff
@@ -108,7 +109,7 @@ public void beforeTextChanged(final CharSequence arg0, final int arg1, final int
 
         expandableListView.setOnChildClickListener((parent, v, groupPosition, childPosition, id) -> {
             GroupItemsInfo groupItem = (GroupItemsInfo) parent.getExpandableListAdapter().getGroup(groupPosition);
-            ChildItemsInfo childItem = (ChildItemsInfo) parent.getExpandableListAdapter().getChild(groupPosition, childPosition);
+            SearchEngine.FitFile.ContentMatchUnit childItem = (SearchEngine.FitFile.ContentMatchUnit) parent.getExpandableListAdapter().getChild(groupPosition, childPosition);
```

Comment:
Why not just use a `Pair<Integer, String>` in both places?

Reply:
I already thought about it and on my own thought that it's easier to follow if it's explicit with name "linenumber".

However changed this now, and at the end it's not that bad as we don't access it much often.